pkg/istio-agent/agent.go

```diff
@@ -122,6 +122,14 @@ type AgentConfig struct {
 	// ferry Envoy's XDS requests to istiod and responses back to envoy
 	// This flag is temporary until the feature is stabilized.
 	ProxyXDSViaAgent bool
+	// DNSCapture indicates if the XDS proxy has dns capture enabled or not
+	// This option will not be considered if proxyXDSViaAgent is false.
+	DNSCapture string
```

Comment:
seems like it could be a bool?

Reply:
Ah yes.. within the agent it shouldn't matter.